### DISCUSSION TITLE
Performance testing

### DISCUSSION THREAD

**Discussion by Jonas-C** [chakra-ui/panda#2845] [General]
Hi! I'm interesting in doing some perf work on the codebase, but there doesn't seem to be too many benchmarking tests at the time of writing. Is this something you've discussed internally?

**Comment by segunadebayo** [comment]
That would be great. Feel free to create benchmarks using the Vitest bench or any other approach that works for you.

For every perf improvement, it’ll be great to see concrete number for the ops/sec.

**Comment by Jonas-C** [comment]
Agreed! Maybe we should consider setting up a ci structure for this to catch potential regressions? That might be a bit above my paygrade, though 🙃 

Being able to perf-test this against actual browsers (playwright, maybe)  as well as against node would be great. AFAIK the vitest bench only targets node.
